sanity-io/structured-content-2022
# Session details page

# Session details page

## Intent

Page for individual Sessions, e.g. [Workshop Lunch][session]

## Description

We've scoped out the ["Session w/1 or 2 speakers"](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=4476%3A16431) variant out of this PR; this PR only tackles the ["Session w/3+ speakers" sketches](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=4476%3A16536).

## Testing this PR

1. Open a session, preferably one tied to a Speaker, e.g. [Workshop Lunch][session]
2. Verify that the page renders all fields correctly and appears in accordance with [the corresponding Figma sketches](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=4476%3A16536) for all supported viewports

[session]: https://structured-content-2022-web-git-session-details-page.sanity.build/program/workshop-lunch
